doc/Variables_Glossary.md

```diff
@@ -0,0 +1,94 @@
+# OMNECT_ADU_DEVICEPROPERTIES_COMPATIBILITY_ID
+Used to configure compatibility between online updates. E.g. if you increase from `0` to `1` an online update via omnect portal is not possible.
+You have to reflash the device with the new image version containing OMNECT_ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "1".
+Part of `compatPropertyNames`, see https://learn.microsoft.com/en-us/azure/iot-hub-device-update/device-update-plug-and-play.
+
+
+# OMNECT_ADU_DEVICEPROPERTIES_MANUFACTURER
+Sets `ADUC_DEVICEPROPERTIES_MANUFACTURER` for iot-hub-device-update. See https://learn.microsoft.com/en-us/azure/iot-hub-device-update/device-update-plug-and-play#device-properties.
+
+# OMNECT_ADU_DEVICEPROPERTIES_MODEL
+Sets `ADUC_DEVICEPROPERTIES_MODEL` for iot-hub-device-update. See https://learn.microsoft.com/en-us/azure/iot-hub-device-update/device-update-plug-and-play#device-properties.
+
+# OMNECT_ADU_MANUFACTURER
+Sets `ADUC_DEVICEINFO_MANUFACTURER` for iot-hub-device-update. See https://learn.microsoft.com/en-us/azure/iot-hub-device-update/device-update-configuration-file (aduc_manufacturer).
+
+# OMNECT_ADU_MODEL
+Sets `ADUC_DEVICEINFO_MODEL` for iot-hub-device-update. See https://learn.microsoft.com/en-us/azure/iot-hub-device-update/device-update-configuration-file (aduc_model).
+
+# OMNECT_BOOT_SCR_NAME
+Name of u-boot bootscript to be generated for devices using u-boot as bootloader. Default is "boot.scr".
+
+# OMNECT_BOOTLOADER_CHECKSUM_COMPATIBLE
+See https://github.com/omnect/meta-omnect/blob/main/classes/omnect_bootloader_versioning.bbclass.
+
+# OMNECT_BOOTLOADER_CHECKSUM_EXPECTED
+See https://github.com/omnect/meta-omnect/blob/main/classes/omnect_bootloader_versioning.bbclass.
+
+# OMNECT_BOOTLOADER_CHECKSUM_FILES
+See https://github.com/omnect/meta-omnect/blob/main/classes/omnect_bootloader_versioning.bbclass.
+
+# OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE
+See https://github.com/omnect/meta-omnect/blob/main/classes/omnect_bootloader_versioning.bbclass.
+
+# OMNECT_BOOTLOADER_VERSION_CHECK_DISABLE
+See https://github.com/omnect/meta-omnect/blob/main/classes/omnect_bootloader_versioning.bbclass.
+
+# OMNECT_BUILD_NUMBER
+`OMNECT_BUILD_NUMBER` is appended to the upstream yocto version and thus part of [`DISTRO_VERSION`](https://docs.yoctoproject.org/ref-manual/variables.html#term-DISTRO_VERSION).
+
+# OMNECT_DEVEL_TOOLS
+List of tools which get installed into a omnect-os devel image.
+
+# OMNECT_FDT_LOAD_NAME
+Name of u-boot device-tree bootscript to be generated for devices using u-boot as bootloader. Default is "fdt-load.scr".
+
+# OMNECT_FLASH_DEVPATH_VAR
+Used in ['flash-mode-1'](https://github.com/omnect/meta-omnect?tab=readme-ov-file#flash-mode-1) initrd script. Contains the path to the target flash device.
+
+# OMNECT_FLASH_MODE
```

Comment:
@JanZachmann  @HarryWaschkeit 
OMNECT_FLASH_* variables are local variables to the flash-mode initrd scritps. does it provide any additional value to document them in a variables glossary?

Reply:
just to be clear, does that mean you propose to only list OMNECT_BOOTLOADER_EMBEDDED_VERSION_* as well instead all of them, or only OMNECT_FLASH_* ?

Imho OMNECT_FLASH_* doesnt make really sense, since it doesn't have a global image impact, if i set it. 
It's more like a local varaible to flash-mode, where it doesn't make sense to set it in conf files at all.

@JanZachmann @HarryWaschkeit imho: to be discussed

Reply:
we decided to non include OMNECT_FLASH_* vars here, since they are local to the initramfs scripts.

Reply:
done

Reply:
just detected, that meanwhile there is one config variable `OMNECT_FLASH_MODE_2_DIRECT_FLASHING` which we have to document here.